obj/objects.h

```diff
@@ -36,7 +38,7 @@ class Sphere : public Object {
   core::Vec3 normal(const core::Vec3 &surfacePoint) const override;
   [[nodiscard]] bool hitObject(const core::Ray &ray, float freq,
                                core::RayHitData *hitData) override;
-
+  [[nodiscard]] bool isVecInside(const core::Vec3 &vec) const;
```

Comment:
BTW: you do not have to add [[nodiscard]] everywhere, only when it is really important, like hitObject, because depending on the result, hitData may be unusable.

Reply:
ok ;)